tests/qam-minimal/install_update.pm

```diff
@@ -80,6 +81,13 @@ sub run {
 
         capture_state('between', 1);
 
+        # old kernel does not have key of new kernel
+        if (check_var('MACHINE', 'uefi') && get_var('BUILD') =~ /dtb|kernel/) {
```

Comment:
Instead of checking `BUILD`, it'd be safer to search incident repos for kernel packages.

Reply:
It would be better, but there are updates with specific build names.
IMO filtering repo packages is more complicated with same result.

Reply:
Those build names can change unpredictably on new SLE releases. Searching the repos does not seem complicated to me at all:

    script_run("zypper -n search kernel -r $repo") == 0

Reply:
True, I had complicated idea in head ...
Thanks, will search for kernel in incident repos.